test/scholar/metrics_test.exs

```diff
@@ -1,5 +1,22 @@
 defmodule Scholar.MetricsTest do
   use ExUnit.Case, async: true
 
+  alias Scholar.Metrics
   doctest Scholar.Metrics
+
+  @y_true Nx.tensor([1.0, 1.0, 2.0, 2.0, 3.0, 3.0], type: {:f, 32})
+  @y_pred Nx.tensor([1.0, 2.0, 1.0, 3.0, 3.0, 3.0], type: {:f, 32})
+  @cm Nx.tensor(
+        [
+          [1, 1, 0],
+          [1, 0, 1],
+          [0, 0, 2]
+        ],
+        type: {:s, 64}
+      )
```

Comment:
You can cover this with a doctest by adding the result below

```
iex> Scholar.Metrics.confusion_matrix(y_true, y_pred)
```

in function docs. See the other metrics :)